doc/source/release-notes/Percona-Server-5.7.26-29.rst

```diff
@@ -0,0 +1,105 @@
+and from the :doc:`Percona Software Repositories </installation>`.
+	
+This release is based on `MySQL 5.7.26
+<http://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-26.html>`_
+and includes all the bug fixes in it. |Percona Server| |release| is
+now the current GA (Generally Available) release in the 5.7 series.
+	
+New Features
+============
+
+- New :variable:`Audit_log_buffer_size_overflow` status variable has been
+  implemented to track when an :ref:`audit_log_plugin` entry was either
+  dropped or written directly to the file due to its size being bigger
+  than :variable:`audit_log_buffer_size` variable.
+
+Bugs Fixed
+==========
+
+- TokuDB storage engine would assert on load when used with jemalloc 5.x.
+  Bug fixed :psbug:`5406`
+
+- a read-write workload on compressed InnoDB tables could cause an assertion
+  error. Bug fixed :psbug:`3581`.
+	
+- using TokuDB or MyRocks native partitioning and ``index_merge`` could lead to
+  a server crash. Bugs fixed :psbug:`5206`, :psbug:`5562`.
+
+- a stack buffer overrun could happen if the redo log encryption with
+  key rotation was enabled. Bug fixed :psbug:`5305`.
+
+- TokuDB and MyRocks native partitioning handler objects were allocated from a 
+  wrong memory allocator. Memory was released only on shutdown and concurrent
+  access to global memory allocator caused memory corruptions and therefore
+  crashes. Bugs fixed :psbug:`5508`, :psbug:`5525`.
+
+- redo log encryption wasn't working. Bug fixed :psbug:`5547`. 
+
+- if there are multiple row versions in InnoDB, reading one row from PK may
+  have O(N) complexity and reading from secondary keys may have O(N^2)
+  complexity. Bugs fixed :psbug:`4712`, :psbug:`5450` (upstream :mysqlbug:`84958`).
+
+- setting the :variable:`log_slow_verbosity` to include ``innodb`` value and
+  enabling the :variable:`slow_query_log` could lead to a server crash.
+  Bug fixed :psbug:`4933`.
+
+- the page cleaner could sleep for long time when the system clock was adjusted
+  to an earlier point in time. Bug fixed :psbug:`5221` (upstream :mysqlbug:`93708`). 
+
+- executing ``SHOW BINLOG EVENT`` from an invalid position could result in a
+  segmentation fault on 32bit machines. Bug fixed :psbug:`5243`.
+
+- ``BLOB`` entries in the binary log could become corrupted
+  in case when a database with ``Blackhole`` tables served as an
+  intermediate binary log server in a replication chain. Bug fixed
+  :psbug:`5353` (upstream :mysqlbug:`93917`).
+
+- when :ref:`audit_log_plugin` was enabled, the server could use a lot of memory when
+  handling large queries.  Bug fixed :psbug:`5395`.
+
+- :ref:`changed_page_tracking` was missing pages changed by the in-place DDL.
+  Bug fixed :psbug:`5447`.
+
+- :variable:`innodb_encrypt_tables` variable accepted ``FORCE`` option only
+  as a string. Bug fixed :psbug:`5538`.
+
+- Redo log encryption would fail to decrypt pages with a certain bit set
```

Comment:
Enabling redo log encryption and changed page tracking together would result in the error log flooded with decryption errors.